testapp/views.py

```diff
@@ -195,5 +204,36 @@ def yelp_call(keyTerm, location):
         )
     return returnData
 
+def get_facebook_info(user):
+    access_token = SocialToken.objects.filter(account__user=user, account__provider='facebook').first()
+    graph = facebook.GraphAPI(access_token=access_token, version=2.10)
+    fb_user_info = graph.get_object(id='me', fields='name, location, hometown, work, likes, friends, education, religion, birthday, age_range')
+    fb_info = ['name', 'location', 'hometown', 'friends', 'education', 'birthday', 'religion', 'age_range']
+    fb_dict = {}
+    user_info = UserInfo()
+    for k in fb_user_info:
+        if k in fb_info:
+            if k == 'education':
+                temp = fb_user_info['education']
+                user_info.education = temp[-1]['school']['name']
+            if k == 'location':
+                user_info.location = fb_user_info['location']['name']
+            if k == 'hometown':
+                user_info.hometown = fb_user_info['hometown']['name']
+            if k == 'friends':
+                user_info.num_friends = fb_user_info['friends']['summary']['total_count']
+            if k == 'name':
+                user_info.name = fb_user_info['name']
+            if k == 'age_range':
+                user_info.age_range = fb_user_info['age_range']['min']
+            if k == 'religion':
+                user_info.religion = fb_user_info['religion']
```

Comment:
I think there might be a better approach to this, like traversing the JSON in some other way but we can come back to this for the sake of time